### PR TITLE
fix: removing support for connect center #671

### PR DESCRIPTION
**Pull request by peternhale**
from https://github.com/forcedotcom/source-deploy-retrieve/pull/671

a copy of pr 671 so tests will be run.

### CODE REVIEW

**Review by mshanemc — APPROVED** [review]

File: METADATA_SUPPORT.md [hunk 1]
```diff
@@ -4,543 +4,542 @@ This list compares metadata types found in Salesforce v55 with the [metadata reg
...
+| FuelType                              | ❌      | Not supported, but support could be added                        |
+| FuelTypeSustnUom                      | ❌      | Not supported, but support could be added                        |
+| IncludeEstTaxInQuoteSettings          | ✅      |                                                                  |
+| MfgServiceConsoleSettings             | ✅      |                                                                  |
+| OauthOidcSettings                     | ✅      |                                                                  |
+| PortalDelegablePermissionSet          | ❌      | Not supported, but support could be added                        |
+| ReportingTypeConfig                   | ❌      | Not supported, but support could be added                        |
+| SustainabilityUom                     | ❌      | Not supported, but support could be added                        |
+| SustnUomConversion                    | ❌      | Not supported, but support could be added                        |
+| UserAccessPolicy                      | ❌      | Not supported, but support could be added                        |
```
Comment (gbockus-sf) [inline METADATA_SUPPORT.md:538]: Sorry I'm out of the loop on this one.  Why did the format change so much here?

Comment (gbockus-sf) [reply METADATA_SUPPORT.md:538]: Assuming the format change was intentional and correct we're good to go 👍

**Review by gbockus-sf — APPROVED** [review]
LGTM

File: METADATA_SUPPORT.md (hunk 1 above)
Comment (peternhale) [reply METADATA_SUPPORT.md:538]: I don't know, I'll have a look

**Pull request merged**